ci/builders/web_engine.json

```diff
@@ -21,7 +30,24 @@
         "tests": []
     },
     {
-        "archives": [],
+        "archives": [
+		{
+			"name": "linux_host_debug_unopt",
+			"include_paths": ["out/host_debug_unopt"],
```

Comment:
I wonder if we can go even further and only include `out/host_debug_unopt/dart-sdk`, but I'm also OK with optimizing this later.